### PR TITLE
add top level usage info

### PR DESCRIPTION
**Pull request by jasondias9**
* added top level org usage endpoint info.
* enhanced usage summary endpoint info

### CODE REVIEW

File: source/includes/administration/_usage.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Attributes | &nbsp;
 `usageType`<br/>*string* | Usage type of the record.
 `secondaryType`<br/>*string* | Secondary type of the record.
 
+*Note*: The utilityUsage reported by this API is the total usage recorded for the specified period. The resource commitment is not taken into account for this endpoint but is outlined in the endpoint `/top_level/organizations/{org_id}` below.
```
Comment (SeventhOfNine) [inline source/includes/administration/_usage.md]: 2nd word is not formatted correctly 'utilityUsage.

File: source/includes/administration/_usage.md [hunk 2]
```diff
@@ -65,6 +67,56 @@ Query Parameters | &nbsp;
 `include_sub_orgs`<br/>*boolean* | Include usage summary of all its sub-organizations. Defaults to false.
 `include_cost`<br/>*boolean* | Include the utility cost and service connection pricing id fields. Defaults to true.
 `include_free_usage`<br/>*boolean* | Include all summary records that has no cost associated to it (i.e. utilityCost == 0). Defaults to true.
-`combine_usage_types`<br/>*boolean* | Sums up all the utility cost per organization and service connection. The following fields are removed from the output: `serviceConnectionPricingId`, `usageType`, `secondaryType`, `utilityUsage` 
+`combine_usage_types`<br/>*boolean* | Sums up all the utility cost per organization and service connection. The following fields are removed from the output: `serviceConnectionPricingId`, `usageType`, `secondaryType`, `utilityUsage`
```
Comment (SeventhOfNine) [inline source/includes/administration/_usage.md]: costs ?

File: source/includes/administration/_usage.md [hunk 3]
```diff
@@ -65,6 +67,56 @@ Query Parameters | &nbsp;
...
+
+Attributes | &nbsp;
+---- | -----------
+`resourceCommitmentUsage`<br/>*string* | The amount of usage that was counted toward the resourceCommitmentCapacity defined by the resource commitment.
+
+##### Resource Commitment Usage:
+The usage that is counted toward a pre assigned pool of resources defined by the Resource Commitment of the organization on a specified service connection. The resource commitment usage is capped by the resource commitment capacity which is the total amount of resource allocated to the organization.
+
+##### Utility Usage:
+Usage that burst outside of the allocated resource commitment if one exists. Note that utility usage will only appear in the response when the `actualUsage` > `resourceCommitmentsUsage`. Note that the `resourceCommitmentUsage` will always be less than or equal to the `resourceCommitmentCapacity` which is defined by the commitment.
```
Comment (SeventhOfNine) [inline source/includes/administration/_usage.md]: bursts ? Or is usage plural ?

File: source/includes/administration/_usage.md (hunk 1 above)
Comment (simongodard) [reply source/includes/administration/_usage.md]: It's supposed to correspond to the attribute returned by the API, 6 lines above. You could put some code syntax to make it more obvious: `utilityUsage`.

File: source/includes/administration/_usage.md (hunk 2 above)
Comment (simongodard) [reply source/includes/administration/_usage.md]: I think both could work, but removing the word `all` would make it easier to read.

File: source/includes/administration/_usage.md (hunk 3 above)
Comment (simongodard) [reply source/includes/administration/_usage.md]: Yes, I think so.

**Review by simongodard — COMMENTED** [review]
I wonder if you should repeat the documentation of the query params and return values for the `List top level organization usage summary` API to make it more self-contained?

File: source/includes/administration/_usage.md (hunk 3 above)
Comment (jasondias9) [reply source/includes/administration/_usage.md]: I updated to bursts

**Comment by jasondias9** [comment]
@simongodard Yes I could do it in this way. Writing this doc makes me wonder why this endpoint couldn't have been merged and been query param driven. (maybe its because there are already so many query params).

**Review by SeventhOfNine — APPROVED** [review]
LGTM

**Review by simongodard — APPROVED** [review]

**Pull request merged**